### DISCUSSION TITLE
zip failed check

### DISCUSSION THREAD

**Discussion by jwhitey2035** [kubilus1/autoortho#64] [General]
Is there anyway to bypass the wrapper and just download directly from the github repo?

Consistently getting this error, which when clicking retry, then re-downloads every single zip that had been extracted before it failed (for example, it fails on na_11, so it then wants to redownload everything from 00-11, even though its extracted the zips into the custom scenery folder already).

Also seem to get stuck on an infinite loop of zip failing, making it impossible to ever install the NA pack.

**Comment by kubilus1** [comment]
Yeah you can just access the zips directly from the companion project here: https://github.com/kubilus1/autoortho-scenery/releases
But there are a few steps beyond just extracting the zips. 

I'll probably re-work the logic at some point, but for now I just cut a pre-release with a change that might help.  There is an added switch in the config file (User dir '.autoortho' file) .   Set 'noclean = True' and it should not wipe out the downloads.
